cosv-backend/build.gradle.kts

```diff
@@ -14,4 +14,10 @@ dependencies {
     implementation(libs.spring.security.core)
     implementation(libs.spring.data.jpa)
     implementation(libs.hibernate.jpa21.api)
+
+    // needed for kubernetes configs and secrets to be added as additional configurations
```

Comment:
no need to keep the comment, `because` is enough

Reply:
done